content/en/blog/_posts/2022-12-01-cpumanager-goes-GA.md

```diff
@@ -0,0 +1,67 @@
+finally when the amount of CPU requested by containers is integral the CPU Manager allocates CPUs exclusively.
+
+This simple operational model served the user base pretty well, but as the CPU Manager matured more and more, users started to look at more elaborate use
+cases and how to better support them.
+
+Rather than add more policies, the community realized that pretty much all the novel use cases are some variation of the behavior enabled by the `static`
+CPU Manager policy. Hence, it was decided to add [options to tune the behavior of the static policy](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/2625-cpumanager-policies-thread-placement#proposed-change).
+The options have a varying degree of maturity, like any other Kubernetes feature, and in order to be accepted, each new option provides a backward
+compatible behavior when disabled, and to document how to interact with each other, should they interact at all.
+
+This enabled the kubernetes project to graduate to GA the CPU Manager core component and core CPU allocation algorithms to GA,
+while also enabling a new age of experimentation in this area.
+In kubernetes v1.26, the CPU Manager [supports three different policy options](docs/tasks/administer-cluster/cpu-management-policies.md#static-policy-options):
+
+- `full-pcpus-only` restrict the CPU Manager core allocation algorithm to full physical cores only, reducing the noisy neighbors issues.
+- `distribute-cpus-across-numa` drive the CPU Manager to evenly distribute CPUs across NUMA nodes in cases where more than one NUMA node is required to satisfy the allocation.
+- `align-by-socket` make the CPU Manager consider CPUs aligned at the socket boundary, instead of NUMA Node boundary, when deciding how to allocate CPUs to a container.
+
+### What’s next?
+
+After graduating the main CPU Manager feature, each existing policy option will graduate to GA. There is room for new options to be added,
+but there's also a growing demand for even more flexibility than what the CPU Manager, and its policy options, currently grant.
+
+Conversations are in progress in the community about splitting the CPU Manager and the other resource managers currently compiled in in the kubelet
+in pluggable, independent kubelet plugins. If you are interested in this effort, please join the conversation on sig-node channels (slack, mailing list, weekly meeting)
+to learn about the [direction the community is pursuing](https://docs.google.com/presentation/d/1uIHF_t97WZzIJgvyD75PYu46PRPi62i2/edit#slide=id.p1).
+
+### How can I learn more?
+
+Please check out the
+[documentation](/docs/tasks/administer-cluster/cpu-management-policies/) for the
+`CPU Manager to learn more about it and how it fits in relation to the other kubelet
+resource managers.
+
+### How to get involved?
+
+This feature is driven by the SIG Node community. Please join us to connect with
```

Comment:
Optionally, link to https://github.com/kubernetes/community/blob/master/sig-node/README.md

Reply:
good idea, done!